tools/debug/debugsymbolviewer.py

```diff
@@ -363,5 +373,7 @@ def print_heap_corruption_data(log_file):
 		print_wrong_sp(log_file)
 	elif (addr_type == 2):
                 print_heap_corruption_data(log_file)
+	elif (addr_type == 3):
```

Comment:
Could you add description for 3 as addr_type to the comment in line 356?
`# 2 for corrupted heap node info, 1 for wrong SP info, 0 to print regular stack addresses`